NAMESPACE

```diff
@@ -104,6 +106,7 @@ exportMethods(spectraVariables)
 exportMethods(spectrapply)
 exportMethods(split)
 exportMethods(tic)
+import(BiocParallel)
```

Comment:
could you please change that to a more selective import? We're already importing `BiocParallel::bpparam` and `BiocParallel::SerialParam`, so maybe this is even not needed.

Reply:
Fixed in latest commit.